django/test/testcases.py

```diff
@@ -301,6 +311,19 @@ def __call__(self, result=None):
         """
         self._setup_and_call(result)
 
+    def __getstate__(self):
+        """
+        Make SimpleTestCase pickable for parallel tests using subtests.
+        """
+        attributes = self.__dict__.copy()
+        # _outcome and _subtest cannot be tested on picklebility, since they contain
+        # the TestCase itself, leading to an infinite recursion.
+        if attributes["_outcome"]:
+            attributes["_outcome"] = None
+            attributes["_subtest"] = None
+            return {k: v for k, v in attributes.items() if is_pickable(v)}
+        return attributes
```

Comment:
Can we also avoid the `__dict__` copy when there is no outcome?

```suggestion
        state = super().__getstate__()
        # _outcome and _subtest cannot be tested on picklebility, since they contain
        # the TestCase itself, leading to an infinite recursion.
        if state["_outcome"]:
            pickable_state = {"_outcome": None, "_subTest": None}
            for key, value in state.items():
                if key in pickable_state or not is_pickable(value):
                    continue
                pickable_state[key] = value
            return pickable
        return state
```